build/azure-pipeline.pre-release.yml

```diff
@@ -0,0 +1,49 @@
+# Run on a schedule
+trigger: none
+pr: none
+
+schedules:
+  - cron: '0 9 * * 1-5' # 9am UTC, Monday-Friday (2am PDT, after VS Code pre-release builds which is 11pm PDT)
+    displayName: Nightly Release Schedule
+    always: true
+    branches:
+      include:
+        - main
+
+resources:
+  repositories:
+    - repository: templates
```

Comment:
Details of this yml can be found here https://github.com/microsoft/vscode-engineering